aws_iot: Update documentation according to AWS certification feedback (#9706)
nrfconnect/sdk-nrf: samples/nrf9160/aws_iot/README.rst

        } to AWS IoT broker
        Next data publication in 60 seconds

> To observe incoming messages in the AWS IoT console, navigate to the AWS IoT Core service and
> click on :guilabel:`MQTT test client`.

Reviewer: ```suggestion
To observe incoming messages in the AWS IoT console, navigate to the AWS IoT Core service and
click :guilabel:`MQTT test client`.
```

Reviewer: Not resolved yet, please remove "on".

> You can observe all messages sent from the device by subscribing to the wild card token **#**.

Reviewer: ```suggestion
To observe all messages sent from the device, subscribe to the wild card token **#**.
```

> .. figure:: images/aws_mqtt_test_client.png
>    :alt: AWS IoT Core MQTT test client

Reviewer: ```suggestion
.. figure:: images/aws_mqtt_test_client.png
   :alt: AWS IoT Core MQTT test client

   AWS IoT Core MQTT test client
```

Testing and debugging
=====================

> If you have issues with the sample, refer to the following section :ref:`gs_testing`.

Reviewer: ```suggestion
If you have issues with the sample, refer to :ref:`gs_testing`.
```

        } to AWS IoT broker
        Next data publication in 60 seconds

To observe incoming messages in the AWS IoT console, navigate to the AWS IoT Core service and
click on :guilabel:`MQTT test client`.
To observe messages that are sent from the device, subscribe to the **$aws/things/<client_id>/shadow/update** shadow topic or the wild card token **#**.

> .. figure:: /images/aws_mqtt_test_client.png

Reviewer: ```suggestion
.. figure:: /images/aws_mqtt_test_client.PNG
```
Either change the file extension here to uppercase or change it in the image file to lowercase.
